admin_guide/preventing_node_placement.adoc

```diff
@@ -0,0 +1,35 @@
+= Preventing Specific Node Placement
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+As a cluster administrator, you can prevent users from targeting specific nodes when scheduling pods. Constraining node placement can be grouped with the scheduler and node selectors to control pod placement. 
+
+== Constraining Pod Placement
+
+To prevent users from targeting specific nodes, modify the master configuration file to look similar to the following:
+
+====
+----
+...
+admissionConfig
```

Comment:
This should also mention the `kubernetesMasterConfig`, as in here: https://github.com/openshift/origin/blob/master/pkg/scheduler/admission/podnodeconstraints/doc.go